homework-3-promises-and-worker-threads/task5/index.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+
+async function readFromFile(filename) {
+   return new Promise((resolve,reject)=>{
+       const resultMap = new Map();
+
+       fs.readFile(filename, 'utf8', (err,data)=> {
+           if (err) {
+               reject(err);
+           } else {
+               data.split("\r\n").forEach((res)=> {
+                   resultMap.set(res.split(":")[0], res.split(":")[1])
+               })
+               resolve(resultMap)
+           }
+       })
+   });
+}
+
+readFromFile("keyValues.txt").then((res)=>console.log(res))
```

Comment:
I think it's incorrect response, we need all values in the console). I think response should be like object and we can show it in json format
![image](https://github.com/user-attachments/assets/cce2c88d-7999-4621-95fd-779792784475)